.github/workflows/ci.yml

```diff
@@ -54,7 +54,7 @@ jobs:
 
   # Run the C integration tests.
   c-tests:
-    uses: lf-lang/lingua-franca/.github/workflows/c-tests.yml@master
+    uses: lf-lang/lingua-franca/.github/workflows/c-tests.yml@arduino-support
```

Comment:
What is the process of CI testing this before we merge the branches together? We can't run our custom c-tests.yml file without specifying the branch but we need to merge these together.